jsk_spot_robot/spoteus/spot-interface.l

```diff
@@ -12,7 +12,7 @@
   (let (r)
     (if wait (ros::wait-for-service srvname))
     (setq r (ros::service-call srvname (instance std_srvs::TriggerRequest :init)))
-    (ros::ros-info "Call \"~A\" returns \"~A\"" srvname (send r :message))
+    (ros::ros-debug "Call \"~A\" returns \"~A\"" srvname (send r :message))
```

Comment:
@sktometometo @mqcmd196 

I am not sure whether this is a good modification.
In the case of watchdog demo, this message is very noise in the console...